### DISCUSSION TITLE
[new visual-diff] issues

### DISCUSSION THREAD

**Discussion by vagusX** [ant-design/ant-design#46477] [General]
att

**Comment by Wxh16144** [comment]
<details>
  <summary>问题讨论起源 #46327</summary>
  
  ## Visual Regression Report for PR #46327
> **Target branch:** feature (11fd15e89c7915cefa42f4d1d6014491f813fde9)
> <a href="https://antd-visual-diff.oss-cn-shanghai.aliyuncs.com/pr-46327/visualRegressionReport/report.html" target="_blank">View Full Report</a> 

------------------------
| image name | expected | actual | diff |
| --- | --- | --- | --- |
| color-picker-panel-render.png | ![feature: 11fd15e89c7915cefa42f4d1d6014491f813fde9](https://antd-visual-diff.oss-cn-shanghai.aliyuncs.com/pr-46327/visualRegressionReport/images/base/color-picker-panel-render.png) | ![current: pr-46327](https://antd-visual-diff.oss-cn-shanghai.aliyuncs.com/pr-46327/visualRegressionReport/images/current/color-picker-panel-render.png) | ![diff](https://antd-visual-diff.oss-cn-shanghai.aliyuncs.com/pr-46327/visualRegressionReport/images/diff/color-picker-panel-render.png) |
| color-picker-panel-render.css-var.png | ![feature: 11fd15e89c7915cefa42f4d1d6014491f813fde9](https://antd-visual-diff.oss-cn-shanghai.aliyuncs.com/pr-46327/visualRegressionReport/images/base/color-picker-panel-render.css-var.png) | ![current: pr-46327](https://antd-visual-diff.oss-cn-shanghai.aliyuncs.com/pr-46327/visualRegressionReport/images/current/color-picker-panel-render.css-var.png) | ![diff](https://antd-visual-diff.oss-cn-shanghai.aliyuncs.com/pr-46327/visualRegressionReport/images/diff/color-picker-panel-render.css-var.png) |



</details>
